src/ic-certified-assets/src/lib.rs

```diff
@@ -203,6 +203,18 @@ struct HttpRequest {
     body: ByteBuf,
 }
 
+#[derive(Clone, Debug, CandidType, Deserialize)]
+struct ChunkInfo {
+    chunk_id: ChunkId,
+    total_length: Nat,
```

Comment:
I would rename this maybe to chunk_length

Reply:
and maybe make it u64

Reply:
@mstein-dfinity done :)